### PR TITLE
Changed the PortableTimer impl to Serilog's one

### PR DESCRIPTION
**Pull request by tukaef**
Fixes issue #15

### CODE REVIEW

File: src/InfluxDB.Collector/Platform/PortableTimer.cs [hunk 1]
```diff
@@ -1,81 +1,122 @@
 ﻿using System;
 using System.Threading;
 using System.Threading.Tasks;
+using InfluxDB.Collector.Diagnostics;
 
-namespace InfluxDB.Collector.Platform
```
Comment (optical) [inline src/InfluxDB.Collector/Platform/PortableTimer.cs]: Did you intend to remove the namespace here?

Comment (tukaef) [inline src/InfluxDB.Collector/Platform/PortableTimer.cs]: Whoops, nope.

**Comment by nblumhardt** [comment]
Awesome, looks good, thanks @tukaef 👍 

One minor thing, can we add the license header from the Serilog file please?

**Comment by nblumhardt** [comment]
Ah, sorry, I forgot I can just edit it in, here. Done!

**Pull request merged**